mgradm/shared/templates/migrateScriptTemplate.go

```diff
@@ -119,7 +119,9 @@ $SSH {{ .SourceFqdn }} sh -c "systemctl list-unit-files | grep hub-xmlrpc-api |
 
 echo "Altering configuration for domain resolution..."
 sed 's/report_db_host = {{ .SourceFqdn }}/report_db_host = localhost/' -i /etc/rhn/rhn.conf;
-sed 's/server\.jabber_server/java\.hostname/' -i /etc/rhn/rhn.conf;
+if test -z "$(grep '^java.hostname *=' /etc/rhn/rhn.conf)\"; then
+    sed 's/server\.jabber_server/java\.hostname/' -i /etc/rhn/rhn.conf;
+fi
```

Comment:
I don't think `test -z` is needed. Grep returns 1 if string not found.

```suggestion
if ! grep -q '^java.hostname *=' /etc/rhn/rhn.conf); then
    sed 's/server\.jabber_server/java\.hostname/' -i /etc/rhn/rhn.conf;
fi
```

should be enough, right?